api4/user.go

```diff
@@ -22,6 +22,7 @@ func InitUser() {
 	BaseRoutes.Users.Handle("", ApiHandler(createUser)).Methods("POST")
 	BaseRoutes.Users.Handle("", ApiSessionRequired(getUsers)).Methods("GET")
 	BaseRoutes.Users.Handle("/ids", ApiSessionRequired(getUsersByIds)).Methods("POST")
+	BaseRoutes.Users.Handle("/names", ApiSessionRequired(getUsersByNames)).Methods("POST")
```

Comment:
Per my comment on the API reference PR, can we make the endpoint use `usernames` over `names`?